frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkTypeOptions.tsx

```diff
@@ -62,12 +62,13 @@ const NetworkTypeOptions = (props) => {
   const dynamicContent = _.map(params, (parameter, key) => {
     const validationMsg = _.get(typeParamsData, [key, 'validationMsg'], null);
     const elemType = _.get(parameter, 'type');
+    const value = _.get(typeParamsData, `${key}.value`, '');
```

Comment:
why not use `initValue || ''` here as the default value instead of '' ?
why not init from `value` field in the initilization list, without adding a new field `initValue`  (assuming `value` is a field that was forgotten in the type deceleration but is actually aepected to be (like the missing `vlan` the linter found for us) )